scripts/bug1749479.pl

```diff
@@ -0,0 +1,106 @@
+#!/usr/bin/env perl
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+#
+# This Source Code Form is "Incompatible With Secondary Licenses", as
+# defined by the Mozilla Public License, v. 2.0.
+
+use 5.10.1;
+use strict;
+use warnings;
+use lib qw(. lib local/lib/perl5);
+
+
+use Bugzilla;
+use Bugzilla::Bug;
+use Bugzilla::Constants;
+use Bugzilla::Group;
+use Bugzilla::Search;
+use Getopt::Long;
+
+Bugzilla->usage_mode(USAGE_MODE_CMDLINE);
+
+my $dbh = Bugzilla->dbh;
+
+# Make all changes as the automation user
+my $auto_user = Bugzilla::User->check({name => 'automation@bmo.tld'});
+$auto_user->{groups}       = [Bugzilla::Group->get_all];
+$auto_user->{bless_groups} = [Bugzilla::Group->get_all];
+Bugzilla->set_user($auto_user);
+
+my $query = {
+  o1 => 'isnotempty',
+  f1 => 'regressed_by',
+  f2 => 'keywords',
+  o2 => 'notequals',
+  v2 => 'regression',
+};
```

Comment:
This won't set `cf_has_regression_range` if a bug has the `regression` keyword, but not `cf_has_regression_range`.  eg [Bug 1750444](https://bugzilla.mozilla.org/show_bug.cgi?id=1750444)

Reply:
This would be much easier to read in field-operator-value order.